packages/flutter/test/material/text_field_test.dart

```diff
@@ -4326,14 +4326,14 @@ void main() {
 
       expect(lastCharEndpoint.length, 1);
       // The last character is now on screen.
-      expect(lastCharEndpoint[0].point.dx, moreOrLessEquals(797.3333129882812));
+      expect(lastCharEndpoint[0].point.dx, moreOrLessEquals(798.6666870117188));
```

Comment:
Tests with values like this don't really inspire confidence. This PR is only an update so I suppose there's no reason to fix the problem here. In a future PR we should either provide a rationale for these values (why is this value expected) or replace them with golden image tests.